PorterAndMoon/Controllers/OrderController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using PorterAndMoon.Connections;
+using PorterAndMoon.Models;
+
+namespace PorterAndMoon.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class OrderController : ControllerBase
+    {
+        readonly OrderConnections _connections;
+
+        public OrderController(OrderConnections connections)
+        {
+            _connections = connections;
+        }
+
+        [HttpGet("All")]
+        public ActionResult GetAllOrders()
+        {
+            var orders = _connections.GetAllOrders();
+            return Accepted(orders);
+        }
+
+        [HttpGet("{Id}")]
+        public ActionResult GetOrder(int Id)
+        {
+            var order = _connections.GetSingleOrder(Id);
+            return Accepted(order);
+        }
+
+        [HttpDelete("{Id}")]
```

Comment:
Comment out delete functionality. Orders shouldn't be deleted from the database.